### ISSUE TITLE
standalone application

### ISSUE THREAD

**Issue by jrivero** [mattf96s/QuackDB#90]
can it become a standalone application? similar than visual studio code...

**Comment by jayendranar02** [comment]
I am interested for contributing in this issue, can I?

**Comment by riziles** [comment]
I agree with @jrivero . This is literally the best DuckDB IDE I have seen. Snap it into a desktop with Tauri, Wails or even PyWebview and it would be unbelievably valuable.

**Comment by Stitch10925** [comment]
Ooh, a Docker image would be amazing too

**Comment by mattf96s** [comment]
Hey! Thanks so much for the kind words!

I’ve been busy lately with moving to a new country and starting a new job, so I haven’t been able to dedicate as much time to the project as I’d like. That said, I’m planning to spend some time on it this weekend and would love to collaborate!
